tests/resources/Vsphere-Util.robot

```diff
@@ -36,6 +36,8 @@ Power Off VM OOB
 
 Destroy VM OOB
     [Arguments]  ${vm}
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc object.method -name Destroy_Task -enable %{VCH-NAME}/${vm}
```

Comment:
when I tried this manually, it didn't work on vc. You need to have `%{TEST_DATACENTER}/vm/%{VCH-NAME}/${vm}` . See https://github.com/vmware/vic/pull/6097/files#diff-fa372ce03d7f9d1bc404459cab0adc75R39

Another question:  why don't we re-enable destroy_task if the host_type is esx?

Reply:
@chengwang86 ESX doesn't support {Enable,Disable}Methods